packages/forms/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": "../../tsconfig.build.json",
+  "references": [
+    { "path": "../ui" },
+    { "path": "../phones" },
+    { "path": "../dates" }
+  ],
+  "compilerOptions": {
+    "composite": true,
+    "rootDir": "src"
+  }
+}
```

Comment:
@umidbekkarimov I have created two `tsconfig` here because `exclude` param is excluding tests and some types are not defined in Editor